Vc/common/bitscanintrinsics.h

```diff
@@ -32,7 +32,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #  if Vc_GCC >= 0x40500
      // GCC 4.5.0 introduced _bit_scan_forward / _bit_scan_reverse
 #    include <x86intrin.h>
-#  else
+#  elif !defined(_bit_scan_forward)
```

Comment:
Apple clang has `/Library/Developer/CommandLineTools//usr/lib/clang/11.0.0/include/x86intrin.h`. Shouldn't we use that instead of entering the `#elif` branch here? I.e. is `#if Vc_GCC >= 0x40500` really correct, or should it be `#if Vc_GCC >= 0x40500 || defined(Vc_CLANG) || defined(Vc_APPLECLANG)`?

Reply:
I thought about this, it's the other option. I just don't know which versions of clang have or don't have the macro. Maybe generically defining it if it's not defined is better than both?

Reply:
The point is that even if the header exists, sometimes the macro is not defined. So we can include the header and if the macro is not defined, we define it.

Reply:
Right, but what happens if `x86intrin.h` defines the macro but wasn't included yet? This header would still define the macro and a subsequent `#include <x86intrin.h>` leads to an error?
I think the right fix is to unconditionally include `x86intrin.h` for GCC, Clang and AppleClang before testing `!defined(_bit_scan_forward)`.

Reply:
Yes, I agree. I will update the change. There were a couple of failures in Travis probably due to that. BTW, do you plan to make a new patch release of 1.4 anytime soon?